products/products.js

```diff
@@ -0,0 +1,15 @@
+import { renderPlant } from '../utils.js';
+import { plants } from './data.js'; 
+
+const ul = document.querySelector('#plants');
```

Comment:
You did it! This I feel the most rewarding part when it all comes together and you have a rendered list.